zeebe/atomix/cluster/src/main/java/io/atomix/raft/protocol/ReplicatableJournalRecord.java

```diff
@@ -63,8 +63,13 @@ public String toString() {
         + index
         + ", checksum="
         + checksum
-        + ", serializedJournalRecord="
-        + Arrays.toString(serializedJournalRecord)
+        + ", serializedJournalRecord={rawToString="
```

Comment:
:thought_balloon: I just realized there is no null check, should I add it first?